qml/AddCredential.qml

```diff
@@ -206,36 +206,33 @@ DefaultDialog {
     }
 
     function getAlgoIndex(algo) {
-        if (algo === 'SHA1') {
+        if (algo === null || algo === 'SHA1') {
             return 0
         }
         if (algo === 'SHA256') {
             return 1
         }
-        return false
     }
 
     function getTypeIndex(type) {
-        if (type === 'TOTP') {
+        if (type === null || type === 'TOTP') {
             return 0
         }
         if (type === 'HOTP') {
             return 1
         }
-        return false
     }
 
     function getDigitsIndex(digits) {
```

Comment:
I feel like we should probably `parseInt` here instead...

Reply:
Not following, parse what int?

Reply:
I mean in `getDigitsIndex`, below this message. :) Instead of matching explicitly on string values, I mean.

Reply:
Sorry, still not following :) After the change we are comparing ints (digits) with ints (index) to preselect the correct option in the dropdown.

Reply:
Um, never mind me. Apparently I went colorblind for a bit there. :D